wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -1190,7 +1191,10 @@ class KotlinGenerator private constructor(
           )
         }
         for (annotation in optionAnnotations(field.options)) {
-          addAnnotation(annotation)
+          // adds use-site target for kotlin properties
+          addAnnotation(annotation.toBuilder().useSiteTarget(PROPERTY).build())
+          // adds use-site target for java fields
```

Comment:
https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html

```suggestion
          // Adds use-site target for kotlin properties.
          addAnnotation(annotation.toBuilder().useSiteTarget(PROPERTY).build())
          // Adds use-site target for java fields.
```